modules/codegen-plugin/src/smithy4s/codegen/GenerateSmithyBuild.scala

```diff
@@ -21,14 +21,14 @@ import sbt.Keys._
 import Smithy4sCodegenPlugin.autoImport._
 
 private final case class SmithyBuildData(
-    imports: Seq[String],
-    deps: Seq[String],
-    repos: Seq[String]
+    imports: Set[String],
+    deps: Set[String],
+    repos: Set[String]
```

Comment:
I'd rather we used `scala.collection.immutable.ListSet` instead. The ordering (in particular for the repos) is significant